### DISCUSSION TITLE
Change comma for decimal for USD values

### DISCUSSION THREAD

**Discussion by mrbracchi** [baimard/gestion#110] [Q&A] [answered]
Hello! Thank you for this interesting software. I am playing with it to see if it is something that I could use for light consulting work and am wondering if there is a way to change the use of a comma for a decimal when showing the owed amount? For example, right now if I have a unit price at $60 USD on the quote and invoice it shows 60,000 $US rather than $60.00 which is standard for a US invoice. I think a client would be very confused and upset seeing an invoice they believe is worth $60,000.00 when they expect an invoice for $60.00
___
Google translate to French:
Salut! Merci pour ce logiciel intéressant. Je joue avec pour voir si c'est quelque chose que je pourrais utiliser pour un travail de conseil léger et je me demande s'il existe un moyen de changer l'utilisation d'une virgule pour une décimale lors de l'affichage du montant dû ? Par exemple, en ce moment, si j'ai un prix unitaire de 60 USD sur le devis et la facture, il indique 60 000 USD au lieu de 60.00 USD, ce qui est standard pour une facture américaine. Je pense qu'un client serait très confus et contrarié de voir une facture qui, selon lui, vaut 60 000,00 $ alors qu'il s'attend à une facture de $60.00

**Comment by baimard** [comment, marked as answer]
Thank you, this is a bug I will change this section :) thanks you for reporting !

**Comment by baimard** [comment]
https://github.com/baimard/gestion/issues/111
